src/main/java/com/epam/java/pdp/collectionsHW/LitterIndexes.java

```diff
@@ -0,0 +1,52 @@
+package com.epam.java.pdp.collectionsHW;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
+//Ex1:
+//        input: any string
+//        output: letter indexes in the string
+//        example:
+//        input: "Hello Word"
+//        output: H - [1], e - 2, l - [3, 4]  , o - [5, 8], whitespace - [6], W - [7], r- [9], d - [10]
+//        Nice to have: sort the output by letter ascending
+public class LitterIndexes {
+    public static void main(String[] args) {
```

Comment:
Separate the java program gateway class (class with main method) and business logic (actual calculation). Move it to separate class